bundles/org.openhab.ui/web/src/components/config/controls/script-editor.vue

```diff
@@ -181,7 +181,13 @@ export default {
       if (mode.startsWith('application/javascript') || mode === 'js') return 'text/javascript'
       if (mode === 'application/vnd.openhab.dsl.rule') return 'text/x-java'
       if (mode === 'application/x-groovy' || mode === 'groovy') return 'text/x-groovy'
-      if (mode === 'application/x-python' || mode === 'py') return 'text/x-python'
+      if (mode === 'application/x-python2' || mode === 'py2') {
+        return {
+          name: 'text/x-python',
+          version: 2
```

Comment:
Is the code able to handle this object instead of the standard string?

Reply:
Yes, I have tested this before I submit it 😉 

There are several options that can be specified, just sending a string for `mode` seems to be "the exception" where the string value is then interpreted as `name`. Python 3 is the default, which is why you don't need to send the `version` when you want that.